### ISSUE TITLE
Hocam şu sıkıntının sebebi nedir ?

### ISSUE THREAD

**Issue by isaturk66** [abdurrahmanekr/bana-istedigini-sor#60]
![hata](https://user-images.githubusercontent.com/35339000/43040758-d5b56fa2-8d54-11e8-8844-ed73ed78fa49.png)

**Comment by abdurrahmanekr** [comment]
Merhaba hatayı zaten içinde vermiş. Şöyle yazıyor: "Unable to resolve module 'AccessibilityInfo' from ..(yolu)"  sorununuzu böyle araştırırsanız bir çözüm bulabilirsiniz. Belki aradığınız cevap [burada](https://github.com/facebook/react-native/issues/14209) vardır.
